ssv/spectest/tests/msg_processing_spectest.go

```diff
@@ -19,6 +19,7 @@ type MsgProcessingSpecTest struct {
 	Duty                    *types.Duty
 	Messages                []*types.SSVMessage
 	PostDutyRunnerStateRoot string
+	PostDutyRunnerState     types.Root
```

Comment:
It is not the same thing.
`PostDutyRunnerStateRoot` - is the root
`PostDutyRunnerState` - is the full state

The interface name is something we all agreed on in a previous PR (I think it was actually an existing interface we moved). However, I agree the name is confusing.
On a different PR we can change the name to `types.Rooter` (@moshe-blox recomendation) because it supposed to go with golang conventions

Reply:
> seems weird to have 2 variables for the same things... why do we need it? just have PostDutyRunnerState and get the root from it? or am I missing something?

it's a good point, that we discussed already with @GalRogozinski 
we want to keep the state only, but only after we move using SC for all the tests

Reply:
It's seems redundant to me.. you can get the root from the state and I don't see a reason to have both.

Reply:
approving, just make sure we have an issue opened to remove it later on

Reply:
It was in the scope of one of the issues.
However, following our conversation I decided to make it clearer:
https://github.com/bloxapp/ssv-spec/issues/228